src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_module_constants.h

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
why there is a second rc_rpc_plugin folder under include? Shall it be just include, since include is already under rc_rpc_plugin?
This comment applies to all files in the same folder.